src/encoded/static/components/static-pages/placeholders/MdSortableTable.js

```diff
@@ -0,0 +1,643 @@
+        //standard tables not use "+" for column headers. first convert any "|---|" to "+---+"
+        input = input.replace(/\|\-{3}/g, "+---").replace(/\-{3}\|/g, "---+");
+
+        columnBorder = input.substring(input.indexOf('+') + 1);
+        columnBorder = '+' + columnBorder.substring(0, columnBorder.indexOf('+') + 1);
+
+        input = input.split('|').map(function (v) { return v.trim(); });
+
+        while (input[i++].indexOf(columnBorder) == -1 && input.length > i);
+
+        const headers = input.slice(1, i - 1);
+
+        input = input.slice(i, input.length - 1);
+
+        let obj = {};
+        input.forEach(function (val, i) {
+            if (i % (headers.length + 1) === (headers.length)) {
+                out.push(obj);
+                return obj = {};
+            }
+
+            obj[headers[i % (headers.length + 1)]] = Number.isNaN(parseFloat(val)) ? val : parseFloat(val);
+        });
+        //push the last item
+        if (!_.isEmpty(obj)) {
+            out.push(obj);
+        }
+
+        return out;
+    },
+    /**
+     * https://github.com/stiang/remove-markdown/blob/master/index.js
+     * @param {*} md - markdown input
+     * @param {*} options - remove options
+     */
+    removeMarkdown: (md, options) => {
+        options = options || {};
+        options.listUnicodeChar = options.hasOwnProperty('listUnicodeChar') ? options.listUnicodeChar : false;
+        options.stripListLeaders = options.hasOwnProperty('stripListLeaders') ? options.stripListLeaders : true;
+        options.gfm = options.hasOwnProperty('gfm') ? options.gfm : true;
+        options.useImgAltText = options.hasOwnProperty('useImgAltText') ? options.useImgAltText : true;
+
+        let output = md || '';
+
+        // Remove horizontal rules (stripListHeaders conflict with this rule, which is why it has been moved to the top)
+        output = output.replace(/^(-\s*?|\*\s*?|_\s*?){3,}\s*$/gm, '');
+
+        try {
+            if (options.stripListLeaders) {
+                if (options.listUnicodeChar)
+                    output = output.replace(/^([\s\t]*)([\*\-\+]|\d+\.)\s+/gm, options.listUnicodeChar + ' $1');
+                else
+                    output = output.replace(/^([\s\t]*)([\*\-\+]|\d+\.)\s+/gm, '$1');
+            }
+            if (options.gfm) {
+                output = output
+                    // Header
+                    .replace(/\n={2,}/g, '\n')
+                    // Fenced codeblocks
+                    .replace(/~{3}.*\n/g, '')
+                    // Strikethrough
+                    .replace(/~~/g, '')
+                    // Fenced codeblocks
+                    .replace(/`{3}.*\n/g, '');
+            }
+            output = output
+                // Remove HTML tags
+                .replace(/<[^>]*>/g, '')
+                // Remove setext-style headers
+                .replace(/^[=\-]{2,}\s*$/g, '')
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary escape character: \-. (no-useless-escape)](https://app.codacy.com/app/4dn/fourfront/pullRequest?prid=4682591)

Reply:
It might be worth considering parsing using existing libraries to avoid having too much code to maintain ourselves. We use markdown-to-jsx; theoretically (having not tested or looked into at all), that library might have some built-in scripts that could be re-used somewhat for that. Idk / thinking out loud / not too important unless there's glitches later.

Reply:
@alexkb0009, you are right. But, digged out markdown-to-jsx library and couldn't find a function on purpose. As a final resort, I copied the maybe-duplicate codes.